addons/website_slides/models/slide_channel.py

```diff
@@ -541,7 +541,8 @@ def _filter_add_members(self, target_partners, **member_values):
                 on_invite.check_access_rights('write')
                 on_invite.check_access_rule('write')
             except:
-                pass
+                if self._context.get('raise_on_not_allowed_to_add_members'):
```

Comment:
I think the preferred way of accessing the context is "self.env.context.get" (not sure if it was already available in v14, but it should be).

Reply:
`self.env.context.get` used 798 vs 522 for `self._context.get` so will go with one most used :sweat_smile:

Reply:
I heard somewhere that they are trying to remove the usage of "_context", that's why I suggested not introducing more of them.